### ISSUE TITLE
Hypothalamic adhesion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6101]
**Preferred term label:**
Hypothalamic adhesion

**Synonyms**
Interhypothalamic Adhesion

**Definition (free text, please give PubMed ID)**
An abnormal parenchymal band connecting the medial margins of the left and right hypothalami across the third ventricle.

Comment: The appearance of the interhypothalamic adhesionis quite characteristic, always visible in the sagittal plane as an additional small ovoid structure between the fornices and lamina terminalis, centered near the level of the midbrain cranial margin. Anatomically, this location corresponds to the anterior and tuberalis regions of the periventricular hypothalamus.
PMID:24874532

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012286

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

https://omim.org/entry/618929

 PMID:31585109 5/9

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
